src/script/team/TeamRepository.ts

```diff
@@ -301,12 +325,7 @@ export class TeamRepository extends TypedEventEmitter<Events> {
     }
   }
 
-  async updateTeamMembersByIds(teamEntity: TeamEntity, memberIds: string[] = [], append = false): Promise<void> {
-    const teamId = teamEntity.id;
-    if (!teamId) {
-      return;
-    }
-
+  async updateTeamMembersByIds(teamId: string, memberIds: string[] = [], append = false): Promise<void> {
```

Comment:
this is a simple API simplification: no need to pass the entire team anymore, just the team ID is enough